debug_module.tex

```diff
@@ -139,17 +139,17 @@ \section{Hart States}
 with indexes higher than the first nonexistent one.
 
 Harts are unavailable if they might exist/become available at a later time, or
-if there are other harts with higher indexes than this one. The most common
-example of this state is when a hart is held in reset. Other examples are a
-hart that is temporarily powered down, or one that might be hot-swapped in
-later.
+if there are other harts with higher indexes than this one. Harts may be
+unavailable for a variety of reasons including being reset, temporarily powered
+down, and not being plugged into the system.  No abstract commands can be
+executed while an unavailable hart is selected.
```

Comment:
While a statement is needed that an unavailable hart can't execute abstract commands, I suggest moving this to the section on abstract commands.  In my opinion, it's better not to say too much about abstract commands before they've been properly introduced.

Reply:
Good points. I've addressed them.